session9/src/homework/DisplayingTodaysDate.java

```diff
@@ -0,0 +1,11 @@
+package homework;
+
+import java.time.LocalDate;
+
+public class DisplayingTodaysDate {
+
+    public static void main(String[] args) {
+        LocalDate displayTodaysDate = LocalDate.now();
```

Comment:
this will work fine, but the purpose of the exercise is to create a method that encapsulates this code, so that it can be called from any location in the code (also here in this `main` method).